### DISCUSSION TITLE
How do ratio breakpoints work and what is the difference vs normal breakpoints?

### DISCUSSION THREAD

**Discussion by muchisx** [nolimits4web/swiper#6643] [Q&A]
Hi!

I couldn't find an explanation of what ratio breakpoints are in the docs, can someone explain how they work and the difference between this and normal screen breakpoints?

https://github.com/nolimits4web/swiper/blob/master/demos/381-ratio-breakpoints.html#L43-L61

https://github.com/nolimits4web/swiper/blob/5a5ebb4162666eb44c1a3e09893309d734289a98/demos/381-ratio-breakpoints.html#L43-L61

**Comment by johnste** [comment]
Was looking for an answer to this too.

Looks like it's the width/height ratio of the viewport – e.g. `@1.50` means the breakpoint will be active if `width * 1.5 > height` is true.
